src/pages/nft/class/_classId/index.vue

```diff
@@ -459,16 +459,14 @@ export default {
               console.error(JSON.stringify(err));
             }
           }),
-      ]);
-      if (metadata?.nft_meta_collection_id?.includes('nft_book')) {
-        await store
-          .dispatch('fetchNFTBookInfoByClassId', classId)
+        store
+          .dispatch('lazyFetchNFTBookInfoByClassId', classId)
           .catch(error => {
             if (error.response?.status !== 400) {
```

Comment:
Is there a case that returns 404?

Reply:
The API now returns 400 for cases that it should return 404, should make another PR pair to "fix" it

Reply:
Should we include 404 here, so this will not break when the API is fixed?